autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/feature_preprocessing/SelectRatesClassification.py

```diff
@@ -0,0 +1,123 @@
+import warnings
+from functools import partial
+from typing import Any, Dict, Optional
+
+from ConfigSpace.conditions import NotEqualsCondition
+from ConfigSpace.configuration_space import ConfigurationSpace
+from ConfigSpace.hyperparameters import (
+    CategoricalHyperparameter,
+    UniformFloatHyperparameter,
+)
+
+import numpy as np
+
+from sklearn.base import BaseEstimator
+from sklearn.feature_selection import GenericUnivariateSelect, chi2, f_classif, mutual_info_classif
+
+from autoPyTorch.datasets.base_dataset import BaseDatasetPropertiesType
+from autoPyTorch.pipeline.components.preprocessing.tabular_preprocessing.feature_preprocessing. \
+    base_feature_preprocessor import autoPyTorchFeaturePreprocessingComponent
+from autoPyTorch.utils.common import FitRequirement, HyperparameterSearchSpace, add_hyperparameter, get_hyperparameter
+
+
+class SelectRatesClassification(autoPyTorchFeaturePreprocessingComponent):
+    """
+    Univariate feature selector by selecting the best features based on
+    univariate statistical tests. Tests can be one of 'chi2, 'f_classif', 'mutual_info_classif'
```

Comment:
Do you know the reason why we only take those tests?
If so, could you describe the reasons because it is not obvious to me?

Reply:
there's an easy answer for this. Sklearn only implements these tests.